### ISSUE TITLE
Proper user list

### ISSUE THREAD

**Issue by l-margiela** [cgm616/0rc#5]
We should display currently connected users

**Comment by cgm616** [comment]
This might be useful: https://stackoverflow.com/questions/7080269/javascript-before-leaving-the-page

Idea is that on load, client publishes that it connected. On leave client publishes that it left. This could be open to abuse by clients, by not sending eithe event, but as long as no one cares too much about extreme accuracy this could work

**Comment by l-margiela** [comment]
We should leave the option to make someone visible/invisible.

**Comment by cgm616** [comment]
I agree. Maybe an "enter chatroom" button that adds you before you can send anything. Of course client side people could still send, but at least our client would respect the site.
